Scripts/summaryStatistics/biophysics/density_calculation.py

```diff
@@ -203,7 +203,7 @@ def calculate_colony_density(img, fig_export_path=''):
     
     return density
     
-def main(cells, fig_export_path=''):
+def main(cells, um_pixel_ratio=0.144, fig_export_path=''):
```

Comment:
um_pixel_ratio parameter should be defined as an optional input parameter in the main function of pyabc_cellmodeller.py.
Also, the main function of the modules is not called "main". Please change this to whatever you want but not "main".

Reply:
I think we can leave um_pixel_ratio hard-coded. We can manually adjust for 10x images and otherwise will not be changing.

Reply:
@Ati-74 um_pixel_ratio coded as an optional parameter here. It can be changed when the function is called.